src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,110 @@
+import { useForm } from '@mantine/form';
+import {
+  TextInput,
+  PasswordInput,
+  Button,
+  Title,
+  Card,
+  Anchor,
+  Text,
+} from '@mantine/core';
+import { IconAt, IconLock } from '@tabler/icons-react';
+import classes from './LoginForm.module.scss';
+import { Link } from 'react-router';
+
+export const LoginForm = () => {
+  const form = useForm({
+    initialValues: { email: '', password: '' },
+    validate: {
+      email: (value) => {
+        const trimmed = value.trim();
+        if (trimmed !== value)
+          return 'Уберите пробелы в начале или конце email';
+        if (!value.includes('@')) return 'Email должен содержать @';
+        if (value.split('@').length !== 2)
+          return 'Должен быть ровно один символ @';
+
+        const [, domain] = value.split('@');
+        if (!domain) return 'Укажите домен после @';
+        if (!domain.includes('.'))
+          return 'Домен должен содержать точку (например: example.com)';
+        if (domain.startsWith('.')) return 'Домен не может начинаться с точки';
+        if (domain.endsWith('.')) return 'Домен не может заканчиваться точкой';
+
+        if (!/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/.test(value)) {
+          return 'Некорректный формат email (пример: user@example.com)';
+        }
+        return null;
+      },
```

Comment:
Mantine has a built-in validation method

```suggestion
email: isEmail('Неверный'),
```

Reply:
Done. Add Mantine isEmail